File: WS/C/Q8.c
#include <stdio.h>

int isPrime(int j){
    int count = 0;
    for(int i = 2; i <= j/2; i++){
        if(j%i == 0){
            count = 1;
        }
    }

    if(count == 0){
        return 1;
    }
    else{
        return 0;
    }
}

int main(){

    int n;
    int i = 0, j = 1;
    int sum = 0;
    printf("Enter range : ");
    scanf("%d" , &n);

    while(1){
        j++;
        if(isPrime(j)){
            sum += j;
            i++;
        }

        if(i == n){
            break;
        }
    }

    printf("Sum of first %d prime number : %d \n",n,sum);
    return 0;
}

File: WS/Python/Q8.py
def prime(n):
    for i in range(2,n):
        if(n%i == 0):
            return 0
    return 1

n = int(input("Enter number : "))
i = 2
lst =[]
sum  = 0
while(1):
    if(prime(i)):
        lst.append(i)
        sum += i
        if(len(lst)==n):
            break
    i+=1
    
print("Sum of first", n ,"Prime numbers : ",sum)